addons/overheating/CfgVehicles.hpp

```diff
@@ -66,7 +66,7 @@ class CfgVehicles {
                 };
                 class GVAR(CoolWeaponWithItem) {
                     displayName = CSTRING(CoolWeaponWithItem);
-                    condition = QUOTE(GVAR(enabled) && {isClass (configfile >> 'CfgPatches' >> 'acex_field_rations')});
+                    condition = 'GVAR(enabled) && {[_player, _target] call FUNC(canCoolWeaponWithItem)} && {!(_unit getVariable [QEGVAR(captives,isSurrendering), false])} && {!(_unit getVariable [QEGVAR(captives,isHandcuffed), false])}';
```

Comment:
```suggestion
                    condition = 'GVAR(enabled) && {[_player, _target] call FUNC(canCoolWeaponWithItem)} && {!(_target getVariable [QEGVAR(captives,isSurrendering), false])} && {!(_target getVariable [QEGVAR(captives,isHandcuffed), false])}';
```

Reply:
This line is getting a little long for configs,
we could move it to the func with something like
```

if ((_unit != _player) && {(_unit getVariable [QEGVAR(captives,isSurrendering), false]) || {_unit getVariable [QEGVAR(captives,isHandcuffed), false]}}) exitWith {
    false;
};
```